src/Utils/Helpers.ts

```diff
@@ -98,3 +98,32 @@ export function obtainWorkspaceRoot(): string {
 export interface FileSelector {
   onFileSelected(uri: vscode.Uri|undefined): void;
 }
+
+/**
+ * Show an information message to ask users whether to save the unsaved file
+ * @param filepath a full path to the file
+ * @return rejected promises if users don't save the file.
+ */
+export async function askToSaveFile(filepath: string): Promise<void> {
+  const unsavedDocuments = vscode.workspace.textDocuments.filter(td => {
```

Comment:
Here, a plural name, `unsavedDocuments`, is used. I am wondering if the length of unsavedDocuments[] is expected to be 1 (or 0 if filepath is not the one in workspace documents) and could you explain the reason of using plural name?

Reply:
Well, I initially worried about having multiple unsaved document editor opened. However I didn't checked if the case really is possible. I will check

Reply:
I inferred so because..
- There is `readonly supportsMultipleEditorsPerDocument?` option for custom editors. 
- By experience, I sometimes see that multiple editors are opened for a single file.

Reply:
From https://code.visualstudio.com/api/extension-guides/custom-editors,

```
If the user opens the same resource in another editor group using our custom 
editor—for example by splitting the first editor—the extension's job is simplified. 
In this case, VS Code just calls resolveCustomEditor with the same CustomDocument 
we created when the first editor was opened.
```

/cc @llFreetimell  We may need to consider using `supportsMultipleEditorsPerDocument: True` in cfg editor.